packages/dynamodb-data-mapper/src/DataMapper.ts

```diff
@@ -1121,12 +1121,13 @@ export class DataMapper {
         return getTableName(item, this.tableNamePrefix);
     }
 
-    private async *mapGetBatch<T extends StringToAnyObjectMap>(
+    private async mapGetBatch<T extends StringToAnyObjectMap>(
```

Comment:
Is this change necessary? Seems unrelated to the bugfix on line 1342

Reply:
You are right this change is not necessary.
I reverted it.